atd-vze/src/views/Tools/CreateCrashRecord.js

```diff
@@ -253,16 +205,12 @@ const CreateCrashRecord = ({ client }) => {
           );
 
           const crashVariables = {
-            atd_fatality_count: fatalityCountSum,
-            address_confirmed_primary: primaryAddress,
-            address_confirmed_secondary: secondayAddress,
+            rpt_street_name: primaryStreetName.toUpperCase(),
+            rpt_sec_street_name: secondaryStreetName.toUpperCase(),
             case_id: caseId,
-            crash_date: crashDate,
-            crash_fatal_fl: fatalityCountSum > 0 ? "Y" : "N",
-            crash_id: tempId,
-            crash_time: crashTime,
-            sus_serious_injry_cnt: susSeriousInjuryCountSum,
-            updated_by: user.email,
+            crash_timestamp: crashTimestamp,
```

Comment:
there's a little timezone bug here, because the user-provided timestamp is being stored in UTC. the way that i've handled this in the past, and i think it's sound, is to create a `Date` instance from the user-input string, and then re-stringify it as an ISO stamp:

```javascript
new Date(crashTimestamp).toISOString()
```